airflow/config_templates/default_airflow.cfg

```diff
@@ -980,6 +980,14 @@ pool_metrics_interval = 5.0
 # This is used by the health check in the "/health" endpoint
 scheduler_health_check_threshold = 30
 
+# When you start a scheduler, airflow starts a tiny web server
+# subprocess to serve a health check if this is set to True
+enable_health_check = True
+
+# When you start a scheduler, airflow starts a tiny web server
+# subprocess to serve a health check on this port
+scheduler_health_check_server_port = 8794
```

Comment:
Could you add both of these config options to the `config.yml` file? This will ensure that they are included in the documentation:

https://github.com/apache/airflow/blob/94f2ce9342d995f1d2eb00e6a9444e57c90e4963/airflow/config_templates/config.yml#L1861